phalcon/Mvc/Model.zep

```diff
@@ -199,7 +199,7 @@ abstract class Model extends AbstractInjectionAware implements EntityInterface,
          */
         let records = this->_getRelatedRecords(modelName, method, arguments);
 
-        if records !== null {
```

Comment:
The same here

Reply:
This is from a test which I wrote to see if non existent related records are handled correctly using `Model::__call()`.
```php
$nonExistentPart = $robotPart->getPart(
            [
                'id < 0',
                'order' => 'id DESC',
            ]
        );
```
If you use `!== null` or `empty` and no records are found, it will throw an exception that the method doesn't exist.

`[Phalcon\Mvc\Model\Exception] The method 'getPart' doesn't exist on model 'Phalcon\Test\Models\RobotsParts' `